addon/templates/components/common/bs-form/element/errors.hbs

```diff
@@ -1,3 +1,11 @@
 {{#if show}}
-  <div class={{feedbackClass}}>{{messages.firstObject}}</div>
+  {{#if showMultipleErrors}}
+    <div class="{{feedbackClass}} pre-scrollable">
+      {{#each messages as |message|}}
+        <p>{{message}}</p>
```

Comment:
What about the `<p>` here, I guess this will add some additional margin to the bottom, i.e. increase the distance from the error message to the following form element? I would think that a single message with `showMultipleErrors=true` should look the same as the first message of many messages with `showMultipleErrors=false`. Because of the slightly different markup here, this would not be the case, right?

Maybe wrap each message in its own `<div class={{feedbackClass}}>`

Reply:
Done